shell/platform/darwin/ios/rendering_api_selection.mm

```diff
@@ -12,16 +12,18 @@
 
 #include "flutter/fml/logging.h"
 
+#import "flutter/shell/platform/darwin/common/framework/Headers/FlutterMacros.h"
 #include "flutter/shell/platform/darwin/ios/framework/Source/FlutterMetalLayer.h"
 
+FLUTTER_ASSERT_ARC
+
 namespace flutter {
 
 bool ShouldUseMetalRenderer() {
   bool ios_version_supports_metal = false;
   if (@available(iOS METAL_IOS_VERSION_BASELINE, *)) {
     auto device = MTLCreateSystemDefaultDevice();
```

Comment:
Optional nit: I don't think this meets the style guide guidance on `auto`, since I couldn't tell what the type was; I would give this an explicit type instead.

Reply:
Done.